packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -244,6 +245,72 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
     return validSubmission;
   }, [setError]);
 
+  const handleVisitAttributes = useCallback(
+    async (visitAttributes: { [p: string]: string }, visitUuid: string) => {
+      const existingAttributes = visitToEdit?.attributes?.map((attribute) => attribute.attributeType.uuid) || [];
+      const attributes = Object.entries(visitAttributes).map(([key, value]) => ({
```

Comment:
I'm not sure why we want to remap an object into an array of objects... Isn't it just as easy to iterate over the object directly, i.e., instead of this step line 256 could just be:

```
for (const [attributeType, value] of Object.entries(visitAtttributes)) {

}
```